pkg/issuer/acme/dns/cloudflare/cloudflare_test.go

```diff
@@ -9,12 +9,14 @@ this directory.
 package cloudflare
 
 import (
+	"encoding/json"
```

Comment:
nit: our [code conventions](https://cert-manager.io/docs/contributing/coding-conventions/) for import formatting state this should be split into 3 blocks

```
standard library

external imports

internal imports
```

Reply:
done :)